lib/fab_circular_menu.dart

```diff
@@ -174,9 +188,11 @@ class FabCircularMenuState extends State<FabCircularMenu>
 
           // FAB
           Container(
+            margin: widget.fabMargin,
```

Comment:
I think this might be faulty here? The Container's margin is removed earlier along with the compensatory translation for that margin. If we add a margin here then it margins the button like originally it wasn't intended to. Correct me if I'm wrong. I find it very good idea to simplify the widget stack by removing the original margin and the translation.